scipy/stats/stats.py

```diff
@@ -4275,6 +4277,158 @@ def mannwhitneyu(x, y, use_continuity=True):
     return MannwhitneyuResult(smallu, distributions.norm.sf(z))
 
 
+# TODO: add paired=False, do a signed-rank test if paired=True or y is not provided. See morestats.wilcoxon
+def mww(x, y, correction=True, exact='auto',
+             alternative='two-sided'):
+    """
+    Computes two-sample unpaired Mann-Whitney-Wilcoxon tests.
+
+    Parameters
+    ----------
+    x : array_like
+        The first set of measurements.
+    y : array_like
+        The second set of measurements.
+    correction : bool, optional
+        If True, apply continuity correction by adjusting the Wilcoxon rank
+        statistic by 0.5 towards the mean value when computing the z-statistic.
+        Default is False.
+    exact : bool, optional
+        Whether an exact test should be performed on the data. See notes for
+        default behavior.
+    alternative : string, optional
+        Whether a two-tailed or one-tailed test should be performed on the
+        supplied vectors. Arguments are 'two-tailed', 'less', and 'greater'.
+        Default is 'two-tailed'.
+
+    Returns
+    -------
+    Bunch object with following attributes:
+        statistic : float
+            The test statistic.
+        pvalue : float
+            The pvalue of the test.
+        exact : bool
+            Indicates if an exact pvalue was calculated.
+        alternative : string
+            Describes the alternative hypothesis.
+        u1 : float
+            The U-value corresponding to the set of measurements in x
+        u2 : float
+            The U-value corresponding to the set of measurements in y
+
+    Notes
+    -----
+    Exact tests should be used for smaller sample sizes. Concretely, as
+    len(x) and len(y) increase to and beyond 8, the distribution of U differs
+    negligibly from the normal distribution[1]. The default behavior of this
+    test is to calculate the number of possible sequences for inputs of
+    length(x) and length(y), and to do an exact calculation if the number of
+    possible combinations is <100000. The default behavior may be overridden
+    with the use_exact flag.
```

Comment:
It might be just my ignorance, but from this explanation I still do not understand what the `exact` flag does. Is it exact versus what, approximate? a normal approximation?